htdocs/comm/action/index.php

```diff
@@ -528,7 +528,11 @@
 if ($user->rights->agenda->myactions->create || $user->rights->agenda->allactions->create) {
 	$tmpforcreatebutton = dol_getdate(dol_now(), true);
 
-	$newparam .= '&month='.str_pad($month, 2, "0", STR_PAD_LEFT).'&year='.$tmpforcreatebutton['year'];
+	if (isset($newparam)) {
+		$newparam .= '&month='.str_pad($month, 2, "0", STR_PAD_LEFT).'&year='.$tmpforcreatebutton['year'];
```

Comment:
An easier fix is to keep only one line of code and just add before the if the init of var
$newparam = '';

Reply:
From what I understand we can't : @fappels mentioned $newparam sometimes contains something for him, so we can't set it, erasing whatever was in prior.

Reply:
Also, wouldn't the amperstand at the beginning of the query string cause problems? (like index.php?**&**month...)

Reply:
I was confused by meaning of 'newparam'. Now have found it's: 'Parameters on current URL' so parameter for backtopage and not after backtopage. 
Think best solution is to put 
$newparam = ''; //Parameters on current URL 
at beginning of script to avoid to have 'null' send to function show_day_events.
url like /index.php?&month... seems not to give any issue and shows correct month.

Reply:
Ok